profiler/options.go

```diff
@@ -45,6 +45,9 @@ const (
 	defaultEnv       = "none"
 )
 
+// Length of valid API key
+const optionAPIKeyLength = 32
```

Comment:
We don't need this constant on the global scope, simply use 32 in the check function. It's obvious enough.

Reply:
And here. The comment isn't resolved until it's resolved

Reply:
Agreed and global removed, value inlined into function in ef1b11d